### DISCUSSION TITLE
How to make the migration with sqlx and create the entities with sea-orm?

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-orm#1301] [Q&A] [answered]
Since I'm [unable to create the database with sea-orm](https://github.com/SeaQL/sea-orm/issues/1282) I thought of creating it directly with sqlite code and then create the entities with sea-orm. Is there an example on how to do this?

**Comment by billy1624** [comment, marked as answer]
Hey @JediMaster25, hi there. You can run raw SQL to create the entity. https://www.sea-ql.org/SeaORM/docs/migration/writing-migration/#raw-sql

**Comment by ghost** [comment]
To not complicate it too much, I've rewritten the schema as sqlite DDL, then I've used sqlite browser to turn the DDL into a `db.sqlite3` database. And then I've run
```
sea-orm-cli generate entity -u sqlite://db.sqlite3 -o database/src/entities
```

When they fix it I'll use the sea-orm migration as I was doing before.

**Reply by billy1624** [reply]
Alright, thanks for the updates :)
